packaging/Dockerfile.centos7-nmstate

```diff
@@ -2,10 +2,15 @@ FROM nmstate/centos7-nmstate-base
 
 ARG SOURCE_COMMIT
 
+# dbus-python is special and does not provide egg files, therefore it is not
+# used by pip and needs to be compiled from PyPi. For this the build
+# dependencies are needed.
+# https://bugzilla.redhat.com/show_bug.cgi?id=1654774
 RUN yum -y install \
         python2-pip \
         git \
     && \
+    yum-builddep -y dbus-python && \
```

Comment:
So this is needed to install nmstate using `pip`?
I had to install `make` and `dbus-devel` for this to work, see https://github.com/nmstate/nmstate/pull/298

Reply:
yum-builddep  uses the build dependencies as defined for the dbus-python RPM, so it should ensure that nothing is forgotten and we do not need to track the dependencies by hand.